tests/hf/test_tapering_hf.py

```diff
@@ -412,11 +384,11 @@ def test_cliford(generator, paulix_ops, result):
         ),
     ],
 )
-def test_transform_hamiltonian(symbols, geometry, generator, paulix_ops, paulix_sector, ham_ref):
+def test_transform_hamiltonian(symbols, geometry, generator, paulixops, paulix_sector, ham_ref):
     r"""Test that transform_hamiltonian returns the correct hamiltonian."""
     mol = qml.hf.Molecule(symbols, geometry)
     h = qml.hf.generate_hamiltonian(mol)()
-    ham_calc = transform_hamiltonian(h, generator, paulix_ops, paulix_sector)
+    ham_calc = qml.taper(h, generator, paulixops, paulix_sector)
```

Comment:
Should we use the imported function (`taper`) directly or the `qml.name`?

Reply:
I prefer to use `qml.name` to make sure that these functions remain high-level and the user interface is always `qml.name`.